@here/harp-omv-datasource/lib/OmvDecoder.ts

```diff
@@ -49,6 +49,7 @@ import {
     OmvDecoderOptions,
     OmvFeatureFilterDescription
 } from "./OmvDecoderDefs";
+import { OmvPointOfViewFeatureModifier } from "./OmvPointOfViewFeatureModifier";
```

Comment:
I cannot see this file. Did you forget to commit it?

Reply:
Yes, my mistake... fixed now.